src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -789,4 +827,20 @@ private void buildCcdRequest(String fileName) throws IOException, URISyntaxExcep
         requestContent = objectMapper.readTree(new File(getClass()
                 .getResource(fileName).toURI()));
     }
+
+    private CallbackRequest buildCallbackRequestWithFinalOrderCollection() {
+        Map<String, Object> caseData = new HashMap<>();
+
+        ArrayList finalOrderCollection = new ArrayList<>();
+        finalOrderCollection.add(HearingOrderCollectionData.builder()
+                .hearingOrderDocuments(HearingOrderDocument
+                    .builder()
+                    .uploadDraftDocument(new CaseDocument())
+                    .build())
+                .build());
+
+        caseData.put(FINAL_ORDER_COLLECTION, finalOrderCollection);
+        CaseDetails caseDetails = CaseDetails.builder().id(Long.valueOf(123)).data(caseData).build();
```

Comment:
call buildCallbackRequest() from base method here and update the case data rather than re-creating everything yourself